src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -394,15 +394,28 @@ public List<ModificationInfos> getModificationsInfos(@NonNull List<UUID> uuids)
         return uuids.stream().map(entities::get).filter(Objects::nonNull).map(this::getModificationInfos).toList();
     }
 
+    /**
+     * @param onlyCommonData if true, only returns the basic data common to all the modifications. If false, returns complete modifications
```

Comment:
Could you explain the basic of the data and what do you mean by onlyCommonData? 
It doesn't make sense to me

Reply:
All the network modifications use the `ModificationInfos`. The data inside this class is the common data to all the network modifications.

But they also have specific data extending this data like `BatteryModificationInfos` or `LineCreationInfos`.